e2e-tests/tfm/cypress/e2e/journeys/case-amendments/7. is-using-facility-end-date.spec.js

```diff
@@ -59,16 +59,14 @@ if (Cypress.env('FF_TFM_FACILITY_END_DATE_ENABLED')) {
       amendmentsPage.amendmentCoverEndDateMonthInput().clear().focused().type(dateConstants.todayMonth);
       amendmentsPage.amendmentCoverEndDateYearInput().clear().focused().type(dateConstants.todayYear);
       amendmentsPage.continueAmendment().click();
-      cy.url().should('contain', 'is-using-facility-end-date');
 
       amendmentsPage.isUsingFacilityEndDateYes().should('be.not.checked');
       amendmentsPage.isUsingFacilityEndDateNo().should('be.not.checked');
     });
 
-    it('should return errors when no options are selected', () => {
+    const navigateToIsUsingFacilityEndDatePageGivenPrefilled = () => {
```

Comment:
I dont think I'd expect this function to be defined in between two `it`s - could either keep it in the `context` and have it at the top, or I think my preference is define it as `function navigateToIsUsingFacilityEndDatePageGivenPrefilled({ user, dealId, facilityId })` at the bottom of the file